app/src/routes/v1/submission.ts

```diff
@@ -23,7 +23,7 @@ router.get(
 );
 
 // Submission create draft endpoint
-router.put('/draft', submissionValidator.createDraft, (req: Request, res: Response, next: NextFunction): void => {
```

Comment:
Wondering if the validator isn't needed anymore?

Reply:
I think this got accidentally added in one of the many merges for the `release/housing-intake` form. I recall this validator wasn't meant for this route.